slides/course2122/seminars/DocEn-02

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this file necessary?

Reply:
No but I couldn't delete it, sorry for the confusion